net-misc/lagrange/lagrange-1.6.1.recipe

```diff
@@ -53,7 +57,9 @@ BUILD()
 	cd build
 	cmake .. \
 		-DCMAKE_INSTALL_PREFIX=$appsDir/Lagrange \
-		-DENABLE_MPG123=ON
+		-DENABLE_MPG123=ON \
+		-DENABLE_HARFBUZZ_MINIMAL=OFF \
+		-DENABLE_FRIBIDI_BUILD=OFF
```

Comment:
Any specific reason (not tested here) why they are disabled if you added them to the requirements (I think leaving them out does the same as cmake wouldn't pick them up anyway)?

Reply:
These flags just say to use the system libs (fribidi and harfbuzz) instead of building them from sources.